cfgov/unprocessed/js/modules/multi-select.js

```diff
@@ -0,0 +1,494 @@
+'use strict';
+
+function init() {
+  var _ = function ( elem, opts ) {
+    var self = this;
+
+    this.elem = $( elem );
+    opts = opts || {};
+
+    configure.call( this, {
+      minChars:  3,
+      maxItems:  10,
+      autoFirst: false,
+      filter: _.FILTER_CONTAINS
+    }, opts );
+
+    this.index = -1;
+    this.name = this.elem.name;
+    this.skipBlur = false;
+    this.optionData = [];
+    this.selections = [];
+    this.options = this.elem.options || [];
+    this.placeholder;
+
+    _.all.push( this );
+  };
+
+  _.prototype = {
+    set options( options ) {
+      this.placeholder = this.elem.getAttribute( 'placeholder' );
+      this.sanitizeList( options );
+
+      if ( this.optionData.length > 0 ) {
+        this.populateMarkup();
+        this.bindEvents();
+      }
+    },
+
+    sanitizeList: function( list ) {
+      var self = this;
+      var l = list.length;
+
+      for ( var i = 0; i < l; i++ ) {
+        if ( list[i].dataset.cflabel === undefined ) {
+          this.optionData.push( {
+            value: list[i].value,
+            text:  list[i].text
+          } );
+        } else {
+          if ( !self.placeholder ) {
```

Comment:
`else` could instead be `else if` or you could drop the else block altogether and include `continue;` statement in the preceding if block.